jest.live.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  preset: 'ts-jest',
+  testEnvironment: 'node',
+  rootDir: './grid',
+  runner: 'groups',
+  reporters: ['default', '../dist/reporter.js'],
+  setupFilesAfterEnv: ['../jest/setup.js'],
```

Comment:
Does creating snapshots works, if the `toMatchSnapshot` is always overridden?

Reply:
It's not always overriden, this config is only used in record mode.